api/golang/core/lib/enclaves/enclave_context.go

```diff
@@ -373,7 +363,6 @@ func (enclaveCtx *EnclaveContext) GetServiceContext(serviceIdentifier string) (*
 	return serviceContext, nil
 }
 
-// Docs available at https://docs.kurtosis.com/sdk#getservicecontexts--mapstring--bool---servicecontext-servicecontext
 func (enclaveCtx *EnclaveContext) GetServiceContexts(serviceIdentifiers map[string]bool) (map[services.ServiceName]*services.ServiceContext, error) {
 	getServiceInfoArgs := binding_constructors.NewGetServicesArgs(serviceIdentifiers)
 	response, err := enclaveCtx.client.GetServices(context.Background(), getServiceInfoArgs)
```

Comment:
Would be good to change these functions to accept a context instead of using `context.Background()` silently.
Can also create other methods like `GetServiceContextsWithContext` that take a context. geth does this in its libraries afair.

Reply:
good point